packages/flutter/lib/src/widgets/table.dart

```diff
@@ -370,7 +403,10 @@ class _TableElement extends RenderObjectElement {
 
```

Comment:
In the mount above, we should be able to calculate the right TableSlot and just pass it into `inflateWidget`, no?

Reply:
So, the only remaining problem is the use of `updateChildren` in `update` then? It looks like we could figure out the appropriate slot value for each child we pass to updateChildren right there. Maybe `updateChildren` could take an optional list of slots (with the same length as the newWidgets list) or a function that generates a slot based on the index of the widget - and when provided the slot from that list or function is used instead of the default IndexSlot?

Reply:
https://github.com/flutter/flutter/pull/76375#discussion_r585177455

Reply:
I am not sure I understand what you mean...